app/src/main/java/com/kickstarter/services/KSUri.java

```diff
@@ -42,6 +42,14 @@ public static boolean isKickstarterUri(final @NonNull Uri uri, final @NonNull St
     return uri.getHost().equals(Uri.parse(webEndpoint).getHost());
   }
 
+  public static boolean isKSGraphQLUri(final @NonNull Uri uri, final @NonNull String webEndpoint) {
+    return isKickstarterUri(uri, webEndpoint) && uri.getPath().equals("/graph");
```

Comment:
Jw, do we need the first conditional here? Or do we not need it in `isWebViewUri`? 🤔

Reply:
Great question. I originally had this as `isGraphQLUri`, where it just checks if the path is `/graph`. But, if we were to use this method in isolation, it might not be exactly what they're thinking - e.g. we might try to use `isGraphQLUri` without checking if it is also a kickstarter uri.

Maybe I'm being a bit overprotective - I'm happy to change it back! And as I'm explaining it, I'm feeling that maybe I should... what do you think?

Reply:
Overprotective is good! I was jw if I was missing some context.